.github/workflows/docker.yml

```diff
@@ -33,12 +31,6 @@ on:
            "linuxmint-21.3",
            "linuxmint-22",
            "linuxmint-22.1",
-           "fedora-30",
-           "fedora-31",
-           "fedora-32",
-           "fedora-33",
-           "fedora-34",
-           "fedora-35",
            "fedora-36",
            "fedora-37",
            "fedora-38",
```

Comment:
Fedora 36-39 are all also end of life: https://docs.fedoraproject.org/en-US/releases/eol/, if you want to remove those too.